### PR TITLE
ft(ZENKO-2348): Update blobserver repo

### PR DESCRIPTION
**Pull request by tmacro**
**What does this PR do, and why do we need it?**
Updates the blobserver & jabba image repos to `registry.scality.com/scality`

**Special notes for your reviewers**:
Target branch will be updated when #898 merges

### CODE REVIEW

File: kubernetes/zenko/charts/opal/values.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ blobserver:
         schedule: '*/5 * * * *'
 
   image:
-    repository: registry.scality.com/sf-eng/blobserver
+    repository: registry.scality.com/scality/blobserver
```
Comment (alexanderchan-scality) [inline kubernetes/zenko/charts/opal/values.yaml:76]: should this be `zenko`?

Comment (rahulreddy) [reply kubernetes/zenko/charts/opal/values.yaml:76]: `zenko` can only host public images, so we are using `scality` namespace to host private ones.

**Review by rahulreddy — APPROVED** [review]

**Comment by tmacro** [comment]
@bert-e approve

**Comment by tmacro** [comment]
@bert-e ping

**Review by alexanderchan-scality — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**